frontend/src/core/project/components/ProjectMenu.js

```diff
@@ -139,7 +139,7 @@ export class ProjectMenuBase extends React.Component<Props, State> {
                 className="selector unselectable"
                 onClick={ this.toggleVisibility }
             >
-                <span>{ 'All Projects' }</span>
+                <span>{ <Localized id= 'resource-ResourceMenu--all-projects'>All Projects</Localized> }</span>
```

Comment:
Please check out how to use the `Localized` tag properly:
https://github.com/mozilla/pontoon/tree/master/frontend#localization

```html
<Localized id="ID">
    <span>Bla</span>
</Localized>
```

Also, mind the space after `=`.